test/azure/generated/Lro/models/index.ts

```diff
@@ -3075,8 +3075,7 @@ export type LROsPutAsyncSubResourceResponse = SubProduct & {
 };
 
 /**
- * Contains response data for the deleteProvisioning202Accepted200Succeeded
```

Comment:
Did comment wrapping break? Or is this expected?

Reply:
Comment wrapping didn't break. Previously these comments were being created inside of Templates using the WrapComment() method. This PR changes them to be created by TSBuilder.DocumentationComment(), which uses a 100 column wrap width, instead of the 80 that WrapComment() uses.